README.md

```diff
@@ -518,13 +518,17 @@ make vivado-fpga FPGA_BOARD=pynq-z2 FUSESOC_FLAGS=--flag=use_bscane_xilinx
 
 Only Vivado 2021.2 has been tried.
 
-To program the bitstream, open Vivado,
+To program the bitstream type:
 
 ```
-open --> Hardware Manager --> Open Target --> Autoconnect --> Program Device
+bash vivado-fpga-pgm FPGA_BOARD=pynq-z2
```

Comment:
Can you pls keep in the README both options ? (With gui ans without so that we cover for those FPGA that fails with the script)